composer.json

```diff
@@ -203,5 +203,8 @@
     "shopsys/product-feed-heureka-delivery": "self.version",
     "shopsys/framework": "self.version",
     "shopsys/coding-standards": "self.version"
+  },
```

Comment:
oooh, sorry i didn't notice that
there is already conflict key for codeception-stub above.